lib/state_machines/integrations/active_record.rb

```diff
@@ -24,14 +24,22 @@ module Integrations #:nodoc:
     # 
     # == Actions
     # 
-    # By default, the action that will be invoked when a state is transitioned
-    # is the +save+ action.  This will cause the record to save the changes
```

Comment:
As far as I know, the bang version of the method returns the result or raises an exception vs the non-bang method will just return the result. It would feel strange if one saves the record and the other doesn't.